lms/views/lti/deep_linking.py

```diff
@@ -204,10 +204,7 @@ def _get_content_url(request):
             if value is not None
         }
 
-        if content["type"] == "file":
```

Comment:
We can remove this now. No new assigmetns will use the older format.

We can't remove the `file_id` detection mechaisms in other parts of the code, eg:

https://github.com/hypothesis/lms/blob/fe74ad8f43bc764523f93b848f2849d4f58e4132/lms/services/document_url.py#L66